core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -432,14 +432,20 @@ object H2OContext extends Logging {
         case e: H2OClusterNodeNotReachableException =>
           H2OContext.get().head.stop()
           throw new H2OClusterNodeNotReachableException(
-            s"""External H2O cluster ${conf.h2oCluster.get} - ${conf.cloudName.get} is not reachable, H2OContext has been closed.
+            s"""External H2O cluster ${conf.h2oCluster.get + conf.contextPath.getOrElse("")} - ${conf.cloudName.get} is not reachable,
```

Comment:
Just a question: Is a delimiting `/` part of `conf.h2oCluster.get + conf.contextPath.getOrElse("")`?

Reply:
Good question, yup, it is 👍